app.js

```diff
@@ -0,0 +1,39 @@
+const os = require('os');
+const fs = require('fs');
+const childProcess = require('child_process');
+
+const commands = {
+  unix: 'ps -A -o %cpu,%mem,comm | sort -nr | head -n 1',
+  windows: 'powershell "Get-Process | Sort-Object CPU -Descending | Select-Object -Property Name, CPU, WorkingSet -First 1 | ForEach-Object { $_.Name + \' \' + $_.CPU + \' \' + $_.WorkingSet }"'
+};
+
+const command = os.platform() === 'win32' ? commands.windows : commands.unix;
+
+const saveData = (data) => {
+  setInterval(() => {
+    const time = new Date().getTime();
+    const newLine = `${time}: ${data}`;
+    fs.appendFile('activityMonitor.log', newLine, (err) => {
+      if (err) throw err;
+      console.warn('Updated!');
+    });
+  }, 60 * 1000);
```

Comment:
for some reason, the frequency of write operations to `activityMonitor.log` file is not 1 minute, but less

maybe it's because you have set the interval inside child process?
what you need to do is vice-versa - you need to call the `saveData` once per minute, which is not dependent on the process that is already running

in other words, here what you should've done:
```suggestion
    const time = new Date().getTime();
    const newLine = `${time}: ${data}`;
    fs.appendFile('activityMonitor.log', newLine, (err) => {
      if (err) throw err;
      console.warn('Updated!');
    });
```